recipes/openscenegraph/all/conanfile.py

```diff
@@ -105,44 +103,43 @@ def configure(self):
             del self.options.with_png
             del self.options.with_dcmtk
 
+        if self.options.get_safe("with_asio", False):
+            raise ConanInvalidConfiguration("ASIO support in OSG is broken, see https://github.com/openscenegraph/OpenSceneGraph/issues/921")
+
     def requirements(self):
         if self.options.enable_windowing_system and self.settings.os == "Linux":
             self.requires("xorg/system")
         self.requires("opengl/system")
 
         if self.options.use_fontconfig:
-            self.requires("fontconfig/2.13.91")
+            self.requires("fontconfig/2.13.92")
 
         if self.options.get_safe("with_asio", False):
             # Should these be private requires?
-            self.requires("asio/1.16.0")
-            self.requires("boost/1.73.0")
-        # if self.options.with_collada:
-        #     self.requires("libxml2/2.9.10")
+            self.requires("asio/1.18.1")
+            self.requires("boost/1.75.0")
```

Comment:
this does not make sense 🤔 boost includes asio...

Reply:
Aren't asio vs boost.asio distinct libraries?

Reply:
They are exactly the same =)

Reply:
I'm not sure what the correct path forward is here. Probably depends on when/if osg decides to fix asio support, so I think it makes sense to leave them both here for now.

Reply:
not exactly the same: https://think-async.com/Asio/AsioAndBoostAsio.html

Reply:
Only one should be required.